### ISSUE TITLE
»hanabi-extension@jeffshee.github.io« does not exist

### ISSUE THREAD

**Issue by Teconia** [jeffshee/gnome-ext-hanabi#87]
I installed it. It workes but it was Japanese. So I deinstalled my JP language pack and tried to reinstall it. and, like the title states, it told me, after reinstalling

»hanabi-extension@jeffshee.github.io« existiert nicht
so in english:
»hanabi-extension@jeffshee.github.io« does not exist.

Should I try by installing EN language pack and deinstall DE l.p.? Is it a general bug? Or is it a user error? 

Please tell me what I should do.

**Comment by jeffshee** [comment]
Hi @Teconia ,

I'm not sure what you mean by "language pack". I switched my system language to Deutsch to try to reproduce the issue you mentioned anyway. However, as you can see in the screenshot below, it didn't show Japanese and was using the default language (English). 

![Bildschirmfoto vom 2023-09-21 08-05-23](https://github.com/jeffshee/gnome-ext-hanabi/assets/25530920/cb464873-0832-494a-bda6-985249429ff4)

Please try installing the extension from the beginning. It seems you had accidentally removed something that the extension required. You could also provide some screenshots next time so I can understand you better. 😃

**Comment by Teconia** [comment]
hi again, I tried doing it from the beginning. I did the following steps

$./run.sh uninstall (or remove...am in school rn don't remember)
-- delete the folder and any associated files I could find

-- reinstalling like in the manual and Ubuntu guide on GH

did I miss something? or should I use "git rm" and if yes, how would I do that? It is my first Ubuntu installation and still don't know how to use "git rm".

**Comment by jeffshee** [comment]
I think you just need to delete the cloned repo (the folder created when you use `git clone`), clone the repo again, and then reinstall by using the same method. Once you have done that, you should be able to get the extension working (at least).

>»hanabi-extension@jeffshee.github.io« existiert nicht

If this is shown again, try re-login.

>I installed it. It workes but it was Japanese.

Only after that, we can try to solve this problem. I wonder why you are seeing Japanese here. But first, let's try to get the extension into a working state.

Also, please provide your OS and Shell information using the template below:
```
**OS and Shell Information**
Please provide your OS name and version, GNOME Shell version, and the session type (X11/Wayland).
 - OS: [e.g. Fedora 37]
 - GNOME Shell: [e.g. 43.3]
 - Session type: [e.g. X11/Wayland]
 ```

**Comment by Teconia** [comment]
System information geman, but it is the window in "settings > info":
![image](https://github.com/jeffshee/gnome-ext-hanabi/assets/121028433/6e85abee-ef1f-4179-971d-2d267abd2ac4)

**Comment by Teconia** [comment]
I accidentally solved the issue. It was because it wasn't English. I installed the English LP and put it in playce of Japanese, to use it as second system language. But still, the offer is up, that I translate it to German.

![image](https://github.com/jeffshee/gnome-ext-hanabi/assets/121028433/ac114653-959f-4e35-8052-41b01940745f)
![Eingefügtes Bild censored](https://github.com/jeffshee/gnome-ext-hanabi/assets/121028433/299dbcd9-2fd4-49b1-b640-e1bedff9c341)

I'm terribly sorry if I bothered you or held you up with something.
